TLM/TLM/Patch/_CitizenAI/_ResidentAI/GetVehicleInfoPatch.cs

```diff
@@ -106,12 +106,14 @@ public class GetVehicleInfoPatch {
             bool useTaxi = !useCar && !useBike && randomizer.Int32(100u) < taxiProb;
             bool useElectricCar = false;
 
+            bool forceElectric = false;
             if (useCar) {
                 int electricProb = GetElectricCarProbability(__instance,
                                                              instanceID,
                                                              ref citizenData,
                                                              __instance.m_info.m_agePhase);
                 useElectricCar = randomizer.Int32(100u) < electricProb;
+                forceElectric = useElectricCar && electricProb == 100;
```

Comment:
this is not even marked as NON-STOCK code. had we forgetten to copy code from vanilla?

Reply:
No it's custom code...